cea/technologies/thermal_network/thermal_network_costs.py

```diff
@@ -0,0 +1,563 @@
+    Opex_a_tot = Opex_var + Opex_fixed
+
+    return Capex_a, Opex_a_tot
+
+
+def calc_Ctot_cooling_plants(network_info):
+    """
+    Calculates costs of centralized cooling plants (chillers and cooling towers).
+    :param network_info: an object storing information of the current network
+    :return:
+    """
+
+    # read in plant heat requirement
+    plant_heat_hourly_kWh = pd.read_csv(
+        network_info.locator.get_optimization_network_layout_plant_heat_requirement_file(
+            network_info.network_type, network_info.config.thermal_network_optimization.network_name))
+    # read in number of plants
+    number_of_plants = len(plant_heat_hourly_kWh.columns)
+
+    plant_heat_original_kWh = plant_heat_hourly_kWh.copy()
+    plant_heat_peak_kW_list = plant_heat_hourly_kWh.abs().max(axis=0).values  # calculate peak demand
+    plant_heat_sum_kWh_list = plant_heat_hourly_kWh.abs().sum().values  # calculate aggregated demand
+
+    Opex_var_chiller = 0.0
+    Opex_var_CT = 0.0
+    Capex_a_chiller = 0.0
+    Opex_a_chiller = 0.0
+    Capex_a_CT = 0.0
+    Opex_a_CT = 0.0
+    # calculate cost of chiller heat production and chiller capex and opex
+    for plant_number in range(number_of_plants):  # iterate through all plants
+        if number_of_plants > 1:
+            plant_heat_peak_kW = plant_heat_peak_kW_list[plant_number]
+        else:
+            plant_heat_peak_kW = plant_heat_peak_kW_list[0]
+        plant_heat_yearly_kWh = plant_heat_sum_kWh_list[plant_number]
+        print 'Annual plant heat production:', round(plant_heat_yearly_kWh, 0), '[kWh]'
+        Capex_chiller = 0
+        Opex_fixed_chiller = 0
+        Capex_CT = 0
+        Opex_fixed_CT = 0
+        if plant_heat_peak_kW > 0:  # we have non 0 demand
+            peak_demand_W = plant_heat_peak_kW * 1000  # convert to W
+            print 'Calculating cost of heat production at plant number: ', plant_number
+            if network_info.config.thermal_network_optimization.yearly_cost_calculations:
+                # calculates operation costs with yearly approximation
+                COP_plant = VCCModel.calc_VCC_COP(network_info.config,
+                                                  network_info.full_cooling_systems,
+                                                  centralized=True)
+                Opex_var_chiller += abs(
+                    plant_heat_yearly_kWh) / COP_plant * 1000 * network_info.prices.ELEC_PRICE
+                Opex_var_CT += CTModel.calc_CT_yearly(plant_heat_yearly_kWh)
+
+            else:
+                # calculates operation costs with hourly simulation
+                # Read in building demand
+                building_demand = {}
+                for building in network_info.building_names:
+                    building_demand[building] = pd.read_csv(
+                        network_info.locator.get_demand_results_file(building))
+                for t in range(HOURS_IN_YEAR):
```

Comment:
I think the entire hourly function is redundant since the current model calculating COP_plant does not change over each hour... but we can keep this structure as a placeholder for future addition.
Please:
1. delete Opex_var_CT (since it is already included in COP_plant, also, the heat input is not correct as explained in previous comment)
2. make note that the current model does not give different results when choosing hourly calculation/ or delete the entire session

Reply:
1. Done :)

2. Actually it should give diferent restults because the supplied systems change depending on if we have an hourly or yearly calculation. The yearly function that I will add in my next commit checks not only if a system is "connected" but also if it has a non zero demand over the year. If the demand is always 0 then this system is excluded for the COP calculation. 

In the hourly calculation this is evaluated at each time step. So our system will have a different COP depending on which loads have nonzero demands. in the yearly calculation the COP is the same for the entrie year.

Reply:
So the COP of the current hourly calculation only changes if the supplied systems are different between each hour, is that right? 
It is independent of the outdoor temperature at the moment?
That is okay for me, since the model is quite rough for the moment anyways, however, it should probably be documented.

Reply:
Currently in both the hourly and the yearly case the COP is a function of t`he yearly average outdoor temperature (in the calc_VCC_COP function).

For the hourly function I will try to clarify:
The supplied systems which are either defined by the optimization or given as a user input from the config file are the systems which we supply with a centralized system. The yearly function checks if one of these systems has an aggregated demand of zero over the year - if it does, the COP is adapted to ignore that system. 
The hourly function does this for every hour. So depending on the demand of that hour, the COP will change. 
E.g. supplied systems = ahu, aru:
t = 10 we have demand for ahu and aru, so the COP is calculated using ahu and aru
t = 11 we only have ahu demand, so COP is calculated using ahu only
t = 12 we only have aru demand, so COP is calculated using aru only
... etc.

I will clarify this in the function description